Лаб. работа 1/задача2.py

```diff
@@ -0,0 +1,11 @@
+inf_V = 1,44 #Информационный объем дискеты в Мб
+sheets = 100 #Количество страниц в книге
+N_strok = 50 #Число строк на странице
+N_simb = 25 #Количество символов в строке
+kod = 4 #Для хранения кода одного символа нужно 4 байта.
+
+book_bite = kod * N_simb * N_strok * sheets #объем одной книги в байтах
+inf_V_bite = inf_V * 1024 * 1024
+all_books = inf_V_bite / book_bite
```

Comment:
Когда мы ищем количество элементов, которые точно поместятся в исходный объем, то более верным решением будет использовать математически оператор - целочисленный остаток от деления `//`, при этом иногда в процессе работы, код выдает это значение в качестве значения с плавающей запятой `float`. Чтобы этого избежать и выдать действительно целочисленное значение, используется метод `int()`
 Тогда строка 9 вашего кода будет выглядеть следующим образом:
```suggestion
all_books = int(inf_v_bite // book_bite)
```